src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -39,6 +39,5 @@
     <include file="db/changelog/db.changelog-RDM-8463.xml"/>
     <include file="db/changelog/db.changelog-RDM-9287.xml"/>
     <include file="db/changelog/db.changelog-RDM-9505.xml"/>
-    <include file="db/changelog/db.changelog-RDM-9052.xml"/>
```

Comment:
just removing the file won't work on environments were the changelog has already been applied, for example Demo. For that case you'd need to explicitly introduce a new changelog that undo the changed introduced by `changelog-RDM-9052`. And which runs only if `changelog-RDM-9052` has run already.